docs/_includes/tests/carousel.html

```diff
@@ -164,3 +164,33 @@ <h4 class="clipped" aria-live="polite">
         </div>
     </div>
 </div>
+
+<h2>Carousel with fade</h2>
+<div class="demo">
+    <div class="demo__inner">
+        <div class="carousel">
+            <div class="carousel__container">
+                <button class="carousel__control carousel__control--prev" aria-label="Previous Slide - Top Products">
+                    <svg aria-hidden="true" class="icon icon--carousel-prev" focusable="false" >
+                        <use xlink:href="#icon-carousel-prev"></use>
+                    </svg>
+                </button>
+                <ul class="carousel__list carousel__fade-right">
```

Comment:
Should this be `carousel__list--fade` instead of `carousel__fade-right`? i.e. a modifier instead of another child block.